### DISCUSSION TITLE
First!

### DISCUSSION THREAD

**Discussion by musnit** [rainbow-me/rainbowkit#314] [Feedback]
https://app.spinamp.xyz/listen -> Click Collection

Super easy to integrate, especially with wagmi <3.

2 issues:
 - Not sure exactly why, but wrapping our app with WagmiProvider and RainbowKitProvider and using ConnectButton.Custom affected the height, so I needed to add extra css of minHeight: '100vh' to fix it. I'd expect it to not impact the UI at all.
 - In our app, we remove and unmount the Connect button (ConnectButton.Custom) as soon as the connection is done. This results in "Warning: Can't perform a React state update on an unmounted component.". I presume this is because the ConnectButton.Custom is expecting some update to it's component, but our unmount gets the update first and so unmounts the button earlier. I could probably do something different to avoid the error, though it would be nice to have an easier way to call openConnectModal without the ConnectButton.Custom or prevent this issue automatically?

**Comment by musnit** [comment]
^ above issue 1 is due to this div: https://github.com/rainbow-me/rainbowkit/blob/e375b0cc5fb1045fd0cb90568c13f95b4cc3937e/packages/rainbowkit/src/components/RainbowKitProvider/RainbowKitProvider.tsx#L84

maybe the div should have height: 100% ?

**Comment by 0xmkh** [comment]
We don't want to add height: 100% since that might cause some unexpected behaviours in UI. The best solution here is to go with minHeight: 100vh and it's common to do so.
